x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/queries/TextExpansionQueryBuilderTests.java

```diff
@@ -84,9 +86,22 @@ protected Object simulateMethod(Method method, Object[] args) {
     }
 
     @Override
-    protected void doAssertLuceneQuery(TextExpansionQueryBuilder queryBuilder, Query query, SearchExecutionContext context)
-        throws IOException {
-        fail("what should be asserted here?");
+    protected void doAssertLuceneQuery(TextExpansionQueryBuilder queryBuilder, Query query, SearchExecutionContext context) {
+
+        assertThat(query, Matchers.instanceOf(MatchNoDocsQuery.class));
```

Comment:
I see your problem. You need to make sure you provide mappings for hte field name you are testing.

Example: From `RankFeatureQueryBuilderTests`

```
@Override
    protected void initializeAdditionalMappings(MapperService mapperService) throws IOException {
        mapperService.merge(
            "_doc",
            new CompressedXContent(
                Strings.toString(
                    PutMappingRequest.simpleMapping(
                        "my_feature_field",
                        "type=rank_feature",
                        "my_negative_feature_field",
                        "type=rank_feature,positive_score_impact=false",
                        "my_feature_vector_field",
                        "type=rank_features"
                    )
                )
            ),
            MapperService.MergeReason.MAPPING_UPDATE
        );
    }
```

Reply:
Thanks the test is passing now. Fixed in d6c6ee4016b7d786a5cbad958d0c0165674ba55c